modules/imgcodecs/src/grfmt_tiff.cpp

```diff
@@ -312,6 +312,18 @@ bool TiffDecoder::readHeader()
                 result = true;
                 break;
             }
+            case 4:
```

Comment:
Please add 4-bit support to error message in default branch.

Reply:
ok, I've pushed the changed, 4-bit is only supported in palette mode.